big-practice/src/style/components/modal.css

```diff
@@ -0,0 +1,25 @@
+/* START MODAL */
+.modal {
+  display: none; 
+  position: fixed;
+  z-index: 1;
+  padding-top: 100px; 
+  left: 0;
+  top: 0;
+  width: 100%;
```

Comment:
100%?? Full screen??? Pls search all modal on the internet to see design, it not full